layouts/partials/cards/bis_table.html

```diff
@@ -15,19 +15,19 @@
           <div class="col-span-2">
               <div class="text-white font-bold py-3 uppercase tracking-widest">Current Tier</div>
               <div class="grid grid-cols-2 gap-3">
-                  {{ range $bisIndex, $bisName := .bis }}
+                  {{ range first site.Params.maxTierGearsets .bis }}
                       {{ $bisItemClass := "bis-item" }}
                       {{/*  TODO: figure out a better id than the name. Its too long and has to many spaces to be a clean link.  */}}
-                      <div class="link"><a href={{ print "best-in-slot/#" $bisName.name }}>{{ $bisName.name }} »</a></div>
+                      <div class="link"><a href={{ print "best-in-slot/#" .name }}>{{ .name }} »</a></div>
```

Comment:
Ha, I think I actually caught a bug from the previous setup. I think this was supposed to be:

```html
<div class="link"><a href="{{ printf "best-in-slot/#%d" $bisIndex }}">{{ $bisName.name }} »</a></div>
```

So the href would link correctly and push over to the right gearset. So you might need to redo this a bit to get the index – sorry!

Reply:
Alright, maked sense, is fixed!